x-pack/legacy/plugins/siem/public/components/ml/anomaly/use_anomalies_table_data.ts

```diff
@@ -83,27 +88,34 @@ export const useAnomaliesTableData = ({
     latestMs: number
   ) => {
     if (userPermissions && !skip && siemJobs.length > 0) {
-      const data = await anomaliesTableData(
-        {
-          jobIds: siemJobs,
-          criteriaFields: criteriaFieldsInput,
-          aggregationInterval: 'auto',
-          threshold: getThreshold(config, threshold),
-          earliestMs,
-          latestMs,
-          influencers: influencersInput,
-          dateFormatTz: getTimeZone(config),
-          maxRecords: 500,
-          maxExamples: 10,
-        },
-        {
-          'kbn-version': config.kbnVersion,
-        }
-      );
-      setTableData(data);
-      setLoading(false);
+      try {
+        const data = await anomaliesTableData(
+          {
+            jobIds: siemJobs,
+            criteriaFields: criteriaFieldsInput,
+            aggregationInterval: 'auto',
+            threshold: getThreshold(config, threshold),
+            earliestMs,
+            latestMs,
+            influencers: influencersInput,
+            dateFormatTz: getTimeZone(config),
+            maxRecords: 500,
+            maxExamples: 10,
+          },
+          {
+            'kbn-version': config.kbnVersion,
+          }
+        );
+        setTableData(data);
+        setLoading(false);
+      } catch (error) {
+        errorToToaster({ title: i18n.SIEM_TABLE_FETCH_FAILURE, error, dispatchToaster });
+        setLoading(false);
```

Comment:
Would we want to set data to `empty` here on error, or does it makes sense to keep showing the previous data?

Reply:
I will keep it the same behavior as the others which is keep the data and not remove it.